Microsoft.Toolkit.Uwp/Helpers/HttpHelper/HttpHelper.cs

```diff
@@ -104,6 +109,45 @@ public HttpHelper(int poolSize, IHttpFilter httpFilter)
             }
         }
 
+        public async Task<HttpHelperResponse> GetRequestAsync(HttpHelperRequest request, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Could you not use SendRequestAsync  ?

I am not comfortable setting default headers for HttpHelper.Instance's HttpClient instances.